packages/terra-list/tests/wdio/multi-select-list-spec.js

```diff
@@ -0,0 +1,207 @@
+    Terra.should.matchScreenshot('first and second item highlighted');
+
+    it('should unhighlight the second item', () => {
+      browser.click('ul li:nth-child(2)');
+    });
+    Terra.should.matchScreenshot('first item highlighted');
+
+    it('should highlight the third item', () => {
+      browser.keys(['Tab', 'Space']);
+    });
+    Terra.should.matchScreenshot('first and third item highlighted');
+
+    Terra.should.beAccessible();
+  });
+
+  describe('Multi Select List Focus', () => {
+    before(() => {
+      browser.url('/#/raw/tests/terra-list/list/multi-select-list/default-multi-select-list');
+      browser.waitForExist('ul');
+      browser.keys(['Tab', 'Enter', 'Tab', 'Enter', 'Shift', 'Tab', 'Shift']);
+    });
+
+    Terra.should.themeCombinationOfCustomProperties({
+      testName: 'themed',
+      properties: {
+        '--terra-list-item-selected-hover-background-color': '#fdebeb',
+      },
+    });
+  });
+
+  describe('Multi Select List On Change', () => {
+    before(() => browser.url('/#/raw/tests/terra-list/list/multi-select-list/on-change-multi-select-list'));
+
+    it('should highlight and display the first item', () => {
+      browser.keys(['Tab', 'Enter']);
+    });
+    Terra.should.matchScreenshot('selected item(s) 0');
+
+    it('should highlight and display the second item', () => {
+      browser.click('ul li:nth-child(2)');
+    });
+    Terra.should.matchScreenshot('selected item(s) 0,1');
+
+    it('should highlight and display the third item', () => {
+      browser.keys(['Tab', 'Enter']);
+    });
+    Terra.should.matchScreenshot('selected item(s) 0,1,2');
+
+    it('should unhighlight and deselect the second item', () => {
+      // Shift-tab to go back, shift at end to release shift key-down
+      browser.keys(['Shift', 'Tab', 'Enter', 'Shift']);
+    });
+    Terra.should.matchScreenshot('selected item(s) 0,2');
+
+    it('should unhighlight and deselect third item', () => {
+      browser.click('ul li:nth-child(3)');
+    });
+    Terra.should.matchScreenshot('selected item(s) 0 third item unhighlighted');
+
+    Terra.should.beAccessible();
+  });
+
+  describe('Multi Select List On Change with Max Selection', () => {
+    before(() => browser.url('/#/raw/tests/terra-list/list/multi-select-list/on-change-multi-select-list-with-max-selection'));
+
+    it('should highlight and display the first item', () => {
+      browser.keys(['Tab', 'Enter']);
+    });
+    Terra.should.matchScreenshot('selected item(s) 0');
+
+    it('should highlight and display the second item', () => {
+      browser.click('ul li:nth-child(2)');
+    });
+    Terra.should.matchScreenshot('selected item(s) 0,1');
+
+    it('should not highlight and not display the third item', () => {
+      browser.keys(['Tab', 'Enter']);
+    });
+    Terra.should.matchScreenshot('selected item(s) 0,1');
+
+    it('should unhighlight and deselect the second item', () => {
+      browser.addValue('ul li:nth-child(2)', 'Space');
```

Comment:
Is this sending the Space key to the second item? I haven't seen this combination before.

Reply:
Yeah, the screenshots verify that it works. For some reason, I couldn't get 
```javascript
browser.element('ul li:nth-child(2)').keys('Space');
``` 
to work.

Reply:
Interesting! Good thing to know 😄 I wasn't sure if we could send keys to a specific element in wdio till now. Thanks!!